packages/expo-file-system/src/next/FileSystem.ts

```diff
@@ -0,0 +1,27 @@
+import ExpoFileSystem from './ExpoFileSystem';
+import { Path } from './FileSystem.types';
+
+if (!ExpoFileSystem) {
+  console.warn(
+    "No native ExponentFileSystem module found, are you sure the expo-file-system's module is linked properly?"
+  );
+}
```

Comment:
Do we need this check? 👀 `requireNativeModule` from `ExpoFileSystem.ts` would throw if the module is not available.